CODEOWNERS

```diff
@@ -17,7 +17,7 @@ misc/*            @akien-mga
 
 modules/bullet/*     @AndreaCatania
 modules/enet/*       @godotengine/network
-modules/gnative/*    @karroffel
+modules/gdnative/*    @karroffel
```

Comment:
should probably remove a space to make up for the added char so identation remains the same?

Reply:
Indeed.